packages/esm-commons-lib/src/utils/encounter-list-utils.ts

```diff
@@ -28,10 +28,10 @@ export function getObsFromEncounters(encounters, obsConcept) {
   return getObsFromEncounter(filteredEnc, obsConcept);
 }
 
-export function getMultipleObsFromEncounter(encounter, obsConcepts: Array<string>) {
+export function getMultipleObsFromEncounter(encounter, obsConcepts: Array<string>, type?: string) {
   let observations = [];
-  obsConcepts.map((concept) => {
-    const obs = getObsFromEncounter(encounter, concept);
+  obsConcepts.forEach((concept) => {
```

Comment:
You won't need to do this either because it will resolve without getting to this function.